libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationMetricsWrapper.java

```diff
@@ -124,6 +124,54 @@ static void rerouteEvent(SessionState sessionState, RouteProgress routeProgress,
     ));
   }
 
+  static void feedbackEvent(SessionState sessionState, RouteProgress routeProgress, Location location,
+                            String description, String feedbackType, String screenshot) {
+    String upcomingInstruction = null;
+    String previousInstruction = null;
+    String upcomingModifier = null;
+    String previousModifier = null;
+    String upcomingType = null;
+    String upcomingName = null;
+    String previousType = null;
+
+    if (routeProgress.currentLegProgress().upComingStep() != null) {
+      upcomingName = routeProgress.currentLegProgress().upComingStep().getName();
+      if (routeProgress.currentLegProgress().upComingStep().getManeuver() != null) {
+        upcomingInstruction = routeProgress.currentLegProgress().upComingStep().getManeuver().getInstruction();
+        upcomingType = routeProgress.currentLegProgress().upComingStep().getManeuver().getType();
+        upcomingModifier = routeProgress.currentLegProgress().upComingStep().getManeuver().getModifier();
+      }
+    }
+
+    if (routeProgress.currentLegProgress().currentStep().getManeuver() != null) {
+      previousInstruction = routeProgress.currentLegProgress().currentStep().getManeuver().getInstruction();
+      previousType = routeProgress.currentLegProgress().currentStep().getManeuver().getType();
+      previousModifier = routeProgress.currentLegProgress().currentStep().getManeuver().getModifier();
+    }
+
+    Location[] beforeLocations = obtainLocations(sessionState.beforeRerouteLocations());
+
+    Location[] afterLocations = obtainLocations(sessionState.afterRerouteLocations());
+
+    String previousName = routeProgress.currentLegProgress().currentStep().getName();
+
+    MapboxNavigationEvent.buildFeedbackEvent(sdkIdentifier, BuildConfig.MAPBOX_NAVIGATION_VERSION_NAME,
```

Comment:
This is currently doing nothing. We're missing pushing the event:

`MapboxTelemetry.getInstance().pushEvent(...);`